### PR TITLE
update ehcache binary in the articatory and made corresponding ol changes

### PR DESCRIPTION
**Pull request by arunavemulapalli**
ehcache updates

### CODE REVIEW

**Comment by arunavemulapalli** [comment]
#libby
#build

**Comment by arunavemulapalli** [comment]
#libby
#build
#spawn.fullfat.buckets=com.ibm.ws.wssecurity_fat.wsscxf

File: dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml [hunk 1]
```diff
@@ -2544,7 +2544,7 @@
     <dependency>
       <groupId>org.jboss.weld.se</groupId>
       <artifactId>weld-se-core</artifactId>
-      <version>3.1.7.SP1</version>
+      <version>3.1.8.Final</version>
```
Comment (neuwerk) [inline dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml]: Did you make this change?

Comment (neuwerk) [reply dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml]: If you didn't it's probably from a bad merge, and you should remove it.

**Comment by arunavemulapalli** [comment]
#libby
#build

**Review by neuwerk — APPROVED** [review]
Thanks Aruna!

**Comment by arunavemulapalli** [comment]
#build

**Comment by arunavemulapalli** [comment]
#libby

**Comment by arunavemulapalli** [comment]
address #19001

**Pull request merged**